zipkin-receiver-kafka/src/main/scala/com/twitter/zipkin/receiver/kafka/KafkaSpanReceiver.scala

```diff
@@ -17,11 +17,12 @@ object KafkaSpanReceiverFactory {
    *
    * @param zookeeper the zookeeper connect string, ex. 127.0.0.1:2181
    * @param topic  the topic zipkin spans will be consumed from
+   * @param streams the count of consumer threads consuming the topic
    */
-  def factory(zookeeper: String, topic: String) = {
+  def factory(zookeeper: String, topic: String, streams: Int = 1) = {
```

Comment:
just a doubt : so `streams` would be passed in `string`. won't that be an issue in casting. (scala noob)